app/models/rentals.py

```diff
@@ -0,0 +1,59 @@
+from flask import current_app
+from app import db
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime, timedelta
+
+class Rental(db.Model):
+    id = db.Column(db.Integer, primary_key=True, autoincrement=True)
+    customer_id = db.Column(db.Integer, db.ForeignKey('customers.id'), primary_key=True)
+    video_id = db.Column(db.Integer, db.ForeignKey('videos.id'), primary_key=True)
+    checkout_date = db.Column(db.DateTime)
+    due_date = db.Column(db.DateTime)
+    check_in_date = db.Column(db.DateTime, nullable=True, default=None)
+    customer = db.relationship("Customer", back_populates="videos")
+    video = db.relationship("Video", back_populates="customers")
+    __tablename__ = "rentals"
+
+    def check_out_to_json(self):
```

Comment:
These functions are a good start, but I would suggest moving the entire business logic of checking out and checking in a video into the models and out of the routes file.  

That way you have the routes file with 1 job (taking requests and giving responses) and the Rental model can deal with making and returning rentals.   It makes the code more flexible.